### ISSUE TITLE
Add keydown action along with onclick on Learn More and Exit Buttons

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/canvas-app-explorer#116]
When using tab and enter to navigate learn more doesn't do anything. We need to be keyboard accessible on these buttons

https://github.com/tl-its-umich-edu/canvas-app-explorer/blob/5fc86c13a0c289ebbf5b96e58213db2e2fdd4255/frontend/components/Home.jsx#L239

**Comment by ssciolla** [comment]
This was partially resolved by #152, but there are a couple more things that need fixing.

**Comment by ssciolla** [comment]
This should be fully resolved after merging #158. We'll need to make sure any future changes to buttons also use proper instrumentation, but Material UI should handle much of this out of the box.
